workflow-templates/cd.properties.json

```
@@ -0,0 +1,12 @@
+{
+    "name": "Publish a Jenkins plugin with Maven",
+    "description": "Continuous delivery for Jenkins plugins and components. For additional setup see jenkinsci/incremental-tools",
+    "iconName": "jenkins",
+    "categories": [
+        "Deployment",
+        "Maven"
+    ],
+    "filePatterns": [
+        "pom.xml$"
```

Without reading the docs here,

```suggestion
        "^pom[.]xml$"
```

?

---

over the top imo, this was based on the docs, it only matches on the root directory, and this is just for suggesting it...